static/events/date_picker.js

```diff
@@ -0,0 +1,49 @@
+function openDatePicker(event){
+    console.log(event)
+    new Litepicker({
+    element: document.getElementById('date-picker'),
+    singleMode: false,
+    tooltipText: {
+      one: 'night',
+      other: 'nights'
+    },
+    tooltipNumber: (totalDays) => {
+      totalDays - 1;
+    },
+    setup: (picker) => {
+
+    picker.on('show', () => {
+        reset_django_form_field()
+      });
+
+    picker.on('selected', (date1, date2) => {
+      getDateValueFromPicker()
```

Comment:
Naming is incorrect, get should always return a value. For ex: If I say get pen then it means take a pen and give it to me. In this case it is not getting date value instead it is doing some other operation with date value